controllers/twophase/types.go

```diff
@@ -150,7 +151,7 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			return ctrl.Result{}, err
 		}
 
-		if err := applyAction(ctx, r, req, chaos); err != nil {
+		if err := applyAction(ctx, r, req, *duration, chaos); err != nil {
```

Comment:
We cannot set the duration when the chaos starts since the duration is not determined. What if it is paused and the chaos is not resumed? Like a chaos has duration 5m, but it only runs 1m and pauses. After 5m, the pause resumes and starts the next round. In this case, the actual duration is only 1m.

Reply:
We can reset it when we pause the chaos. This field is required by the front-end.

Reply:
Yes. This is a good idea. I agree we should change update this when we pause